jaeger/cmd/uninstall.go

```diff
@@ -34,17 +38,5 @@ func uninstallRunE(ctx context.Context) error {
 		return err
 	}
 
-	resources, err := resource.FetchKubernetesResources(ctx, k8sAPI,
-		metav1.ListOptions{LabelSelector: "linkerd.io/extension=jaeger"},
-	)
-	if err != nil {
-		return err
-	}
-
-	for _, r := range resources {
-		if err := r.RenderResource(os.Stdout); err != nil {
-			return fmt.Errorf("error rendering Kubernetes resource: %v", err)
-		}
-	}
-	return nil
+	return pkgCmd.Uninstall(ctx, k8sAPI, "linkerd.io/extension=jaeger")
```

Comment:
We can copy the same format we have in viz's `uninstall` command here as well same with `multicluster`:
```suggestion
	return pkgCmd.Uninstall(ctx, k8sAPI, fmt.Sprintf("%s=%s", k8s.LinkerdExtensionLabel, JaegerExtensionName))
```

Here is the full diff of the changes:
```diff
diff --git a/jaeger/cmd/uninstall.go b/jaeger/cmd/uninstall.go
index 32e119d8..f03592e0 100644
--- a/jaeger/cmd/uninstall.go
+++ b/jaeger/cmd/uninstall.go
@@ -38,5 +38,5 @@ func uninstallRunE(ctx context.Context) error {
                return err
        }

-       return pkgCmd.Uninstall(ctx, k8sAPI, "linkerd.io/extension=jaeger")
+       return pkgCmd.Uninstall(ctx, k8sAPI, fmt.Sprintf("%s=%s", k8s.LinkerdExtensionLabel, JaegerExtensionName))
 }
diff --git a/multicluster/cmd/uninstall.go b/multicluster/cmd/uninstall.go
index 15e43529..1cef3131 100644
--- a/multicluster/cmd/uninstall.go
+++ b/multicluster/cmd/uninstall.go
@@ -53,7 +53,7 @@ func newMulticlusterUninstallCommand() *cobra.Command {
                                return errors.New(strings.Join(err, "\n"))
                        }

-                       err = pkgCmd.Uninstall(cmd.Context(), k8sAPI, "linkerd.io/extension=multicluster")
+                       err = pkgCmd.Uninstall(cmd.Context(), k8sAPI, fmt.Sprintf("%s=%s", k8s.LinkerdExtensionLabel, MulticlusterExtensionName))
                        if err != nil {
                                fmt.Fprintln(os.Stderr, err)
                                os.Exit(1)
```